src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,59 @@
+package wooteco.subway.ui;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+    
+    @PostMapping
+    public ResponseEntity<LineResponse> createStation(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = lineService.save(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<LineResponse> lineResponses = lineService.findAll();
+        return ResponseEntity.ok().body(lineResponses);
```

Comment:
```suggestion
        return ResponseEntity.ok(lineResponses);
```

사소한 것이긴 하지만, OK 응답의 경우 이렇게 shortcut을 지원하고 있어요!

Reply:
`ok(T body)` 메서드에서 상태를 OK로도 설정하면서 body에 데이터까지 저장하는 것을 지원하는군요!
꿀팁 감사합니다!😊😊